test/test_issue283.py

```diff
@@ -81,16 +81,22 @@ class SomePerson(BaseOtherPerson):
 
 
 # Test cases
-def test_automatic_object_resolution():
+def test_automatic_result_resolution():
```

Comment:
Why is this test getting renamed here?

Reply:
I have renamed both test because I have renamed the original method.

Instead of `_object_resolution`, I have renamed it to `_result_resolution` and in fact used `_object_resolution` for the part that.. resolves objects.

Check https://github.com/neo4j-contrib/neomodel/pull/716/files#diff-5a14848686f658a9fab17bad447b0d3631c13d078bd3bbde13804138e3150e24R267 and https://github.com/neo4j-contrib/neomodel/pull/716/files#diff-5a14848686f658a9fab17bad447b0d3631c13d078bd3bbde13804138e3150e24R233

Therefore, I have renamed the test methods to match with the name of the method it is covering. I can revert that if you want :)